CHANGELOG.md

```diff
@@ -21,11 +21,13 @@
 ### Banners and guidance instructions
 
 * Fixed the crash when scrolling the guidance cards while the orientaion changes. ([#3544](https://github.com/mapbox/mapbox-navigation-ios/pull/3544))
+* A `WayNameView` now can be fully hidden by using `WayNameView.isAlwaysHidden` property. ([#3534](https://github.com/mapbox/mapbox-navigation-ios/pull/3534))
 
 ### CarPlay
 
 * Added the `CarPlayActivity.panningInNavigationMode` case, which allows to track a state when user is panning a map view while actively navigating. ([#3545](https://github.com/mapbox/mapbox-navigation-ios/pull/3545))
 * Fixed an issue that caused the panning dismissal button to stop working on CarPlay. ([#3543](https://github.com/mapbox/mapbox-navigation-ios/pull/3543))
+* Added the `CarPlayMapViewController.wayNameView` and `CarPlayNavigationViewController.wayNameView` properties to show the current road name on CarPlay. `CarPlayNavigationViewController.compassView`, `CarPlayNavigationViewController.speedLimitView` and `CarPlayMapViewController.speedLimitView` are kept as strong references, thus available throughout the lifetime of a parent object. ([#3534](https://github.com/mapbox/mapbox-navigation-ios/pull/3534))
```

Comment:
The parent object in this case would be CarPlayManager, right?

Reply:
By parent object I mean either `CarPlayNavigationViewController` or `CarPlayMapViewController`.